python/lsst/ip/isr/calibType.py

```diff
@@ -188,6 +188,8 @@ def updateMetadata(self, camera=None, detector=None, filterName=None,
                 (self._raftName, self._slotName) = self._detectorName.split("_")
 
         if filterName:
+            # TOD0 DM-28093: I think this whole comment can go away, if we
+            # always use physicalLabel everywhere in ip_isr.
             # If set via:
             # exposure.getInfo().getFilter().getName()
             # then this will hold the abstract filter.
```

Comment:
I'm not sure you can use `physicalLabel` everywhere in `ip_isr` (at least, not at the UI level), but this comment seems out of date regardless.

Reply:
I'd rather not muck with it yet: it might be useful guidance for the implementer of DM-28093.